model/jpa/src/main/resources/META-INF/jpa-changelog-3.4.0.xml

```diff
@@ -47,4 +47,63 @@
         <addPrimaryKey columnNames="CLIENT_ID,VALUE" constraintName="CONSTRAINT_WEB_ORIGINS" tableName="WEB_ORIGINS"/>
     </changeSet>
 
+    <changeSet author="hmlnarik@redhat.com" id="3.4.0-KEYCLOAK-5230">
+
+        <createIndex tableName="FED_USER_ATTRIBUTE" indexName="IDX_FED_USER_ATTRIBUTE">
+            <column name="USER_ID" type="VARCHAR(255)" />
+            <column name="REALM_ID" type="VARCHAR(36)" />
+            <column name="NAME" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_CONSENT" indexName="IDX_FED_USER_CONSENT">
+            <column name="USER_ID" type="VARCHAR(255)" />
+            <column name="CLIENT_ID" type="VARCHAR(36)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_CONSENT" indexName="IDX_FED_USER_CONSENT_RU">
+            <column name="REALM_ID" type="VARCHAR(36)" />
+            <column name="USER_ID" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_CREDENTIAL" indexName="IDX_FED_USER_CREDENTIAL">
+            <column name="USER_ID" type="VARCHAR(255)" />
+            <column name="TYPE" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_CREDENTIAL" indexName="IDX_FED_USER_CREDENTIAL_RU">
+            <column name="REALM_ID" type="VARCHAR(36)" />
+            <column name="USER_ID" type="VARCHAR(255)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_GROUP_MEMBERSHIP" indexName="IDX_FED_USER_GROUP_MEMBERSHIP">
+            <column name="USER_ID" type="VARCHAR(255)" />
+            <column name="GROUP_ID" type="VARCHAR(36)" />
+        </createIndex>
+
+        <createIndex tableName="FED_USER_GROUP_MEMBERSHIP" indexName="IDX_FED_USER_GROUP_MEMBERSHIP_RU">
```

Comment:
indexName IDX_FED_USER_GROUP_MEMBERSHIP_RU is too long on Oracle DB, the limit is 30